doc/src/guides/solving/solve-system-of-inequalities-algebraically.md

```diff
@@ -0,0 +1,229 @@
+# Solve a system of inequalities algebraically
+
+Use SymPy to solve a system of inequalities algebraically. For example, solving
+$x^2 < \pi$, $x > 0$ yields $0 < x < \sqrt{pi}$.
```

Comment:
I would expect a system of inequalities to mean a *multivariate* system of inequalities involving multiple unknowns. Using `solve` like this only works for *univariate* inequalities e.g.:
```
In [2]: solve([x > y**2, y > 0], [x, y])
---------------------------------------------------------------------------
NotImplementedError
```
The fact that this does not handle multivariate inequalities should be made clear in the title